summoner-tui/summoner-tui.cabal

```diff
@@ -0,0 +1,88 @@
+cabal-version:       2.0
+name:                summoner-tui
+version:             0.0.0
+synopsis:            Tool for scaffolding completely configured production Haskell projects using TUI.
+description: :       Tool for scaffolding completely configured production Haskell projects using TUI.
+                     See README.md for details.
+homepage:            https://github.com/kowainik/summoner
+bug-reports:         https://github.com/kowainik/summoner/issues
+license:             MPL-2.0
+license-file:        LICENSE
+author:              Kowainik
+maintainer:          xrom.xkov@gmail.com
+copyright:           2018 Kowainik
+category:            TUI, TUI Tool, Development
+build-type:          Simple
+stability:           stable
+extra-doc-files:     ../README.md
+                   , CHANGELOG.md
+tested-with:         GHC == 8.2.2
+                   , GHC == 8.4.4
+                   , GHC == 8.6.2
+
+source-repository head
+  type:     git
+  location: git@github.com:kowainik/summoner.git
+
+library
+  hs-source-dirs:      src
+  exposed-modules:     Summoner.Tui
+                         Summoner.Tui.Field
+                         Summoner.Tui.Form
+                         Summoner.Tui.GroupBorder
+                         Summoner.Tui.Kit
+                         Summoner.Tui.Validation
+                         Summoner.Tui.Widget
+
+  autogen-modules:     Paths_summoner_tui
+  other-modules:       Paths_summoner_tui
```

Comment:
I now think that we should somehow use this in CLI `--version` option. As we are having different versions for `summoner` and `summoner-tui` we should reflect it in this command as well. Do you think it's doable?